config/confighttp/confighttp.go

```diff
@@ -123,13 +124,30 @@ type HTTPServerSettings struct {
 	// CORS needs to be enabled first by providing a non-empty list in CorsOrigins
 	// A wildcard (*) can be used to match any header.
 	CorsHeaders []string `mapstructure:"cors_allowed_headers"`
+
+	// MuxListener is the multiplexing listener for the Endpoint. Through multiplexing, we can
+	// support many sockets on the same port with the same listener.
```

Comment:
Some minor doc clarification:
Same as the other suggestion:
```suggestion
	// MuxListener is a multiplexing listener for the Endpoint. Through multiplexing, 
	// a single listener can support multiple sockets on the same port.
```